.github/workflows/run_tests.yml

```diff
@@ -6,15 +6,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        golang-version: [1.13, 1.14, 1.15, 1.16, 1.17]
+        golang-version: [1.21]
```

Comment:
👍

Reply:
Like that you are going to 1.21, just note that the version number should be wrapped in quotes so it doesn't get interpreted as a number. yaml gotchas...

Reply:
Good observation, though I think this action handles that by converting the float to a string, it's probably a safer approach to have this as a string.